install/Makefile

```diff
@@ -4,34 +4,35 @@ include ../Makefile.os
 
 RELEASE_VERSION ?= latest
 RELEASE_PATH ?= ../strimzi-$(RELEASE_VERSION)/$(PROJECT_NAME)
+HELM_VERSION ?= helm3
```

Comment:
Similarly this should maybe split out into two variables similar to described above